pydy/tests/test_bodies.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+
+SYMPY_VERSION = sm.__version__
+
+
+class TestBodies():
+
+    def test_body_init(self):
+        body = Body('body')
+
+        assert body.name is None
+        assert body.mass == Symbol('mass')
+
+        sys = System(method='joints')
+        sys.add_body(body)
+
+        assert body.name == 'body_1'
+
+        body_length = Length(body.name)
+        assert body_length.symbol == Symbol(body.name + "_length")
```

Comment:
an `OrderedDict` or something should do for a parameter and its constant value pair. If we have require more functionality apart from simply storing the data(params, symbols etc.) we can think of having a separate class, and put the functionality in classmethods.